engine/storage/image/src/org/apache/cloudstack/storage/image/TemplateServiceImpl.java

```diff
@@ -675,6 +695,18 @@ protected Void createTemplateCallback(AsyncCallbackDispatcher<TemplateServiceImp
             return null;
         }
 
+        // Check if OVA contains additional data disks. If yes, create Datadisk templates for each of the additional datadisk present in the OVA
+        if (template.getFormat().equals(ImageFormat.OVA)) {
+            if (!createDataDiskTemplates(template)) {
```

Comment:
is there a way for `createDataDiskTemplates` to return false?

Reply:
Thanks, actually there wasn't. I've now provided a way to return false in case of checked exceptions